src/main/java/lambdastreams/task4/Hello.java

```diff
@@ -0,0 +1,15 @@
+package lambdastreams.task4;
+
+public class Hello {
+    public static void main(String[] args) {
+        StringFunction exclaim = (s) -> s + "! " ;
```

Comment:
Please remove the brackets around the "s" variable

Reply:
Done